http-gateway/pkg/server/gateway_integration_test.go

```diff
@@ -79,40 +82,37 @@ var _ = Describe("HTTP Gateway", func() {
 				message.Headers{server.CorrelationId: msg.Headers()[server.CorrelationId],
 					"Content-Type": []string{"bag/plastic"},
 				})
-			Expect(msg.Headers()["Content-Type"]).To(Equal([]string{"text/solid"}))
-			Expect(msg.Headers()["Not-Propagated-Header"]).To(BeNil())
+			Eventually(msg.Headers()["Content-Type"]).Should(Equal([]string{"text/solid"}))
```

Comment:
Eventually should take a closure rather than a value, since a value will never change. ;-)